vaadin-list-mixin.html

```diff
@@ -166,7 +166,7 @@
         return;
       }
 
-      const condition = item => !item.disabled;
+      const condition = item => !(item.disabled || window.ShadyCSS.getComputedStyleValue(item, 'display') == 'none');
```

Comment:
The code should also work without ShadyCSS.

Reply:
done